CloudFlare/cloudflare.py

```diff
@@ -48,6 +48,14 @@ def call_with_no_auth(self, method, parts,
             return self._call(method, headers, parts,
                               identifier1, identifier2, identifier3,
                               params, data, files)
+        
+        def _auth_headers(self, headers):
```

Comment:
```suggestion
        def _add_auth_headers_to(self, headers):
```

I think it can be add, set, or append, but having only a noun reads weird.